distributed/client.py

```diff
@@ -105,7 +110,7 @@
 NO_DEFAULT_PLACEHOLDER = "_no_default_"
 
 
-def _get_global_client():
+def _get_global_client() -> Client | None:
```

Comment:
correct. 3.10 syntax works on all python versions as long as your version of mypy is recent.
Notable exception: ``cast``, since it is executed at runtime, needs Python 3.10 style annotations wrapped in a string.